### DISCUSSION TITLE
Hi There,

### DISCUSSION THREAD

**Discussion by MyBaBB** [community/community#84507] [New to GitHub]
“I started learning GitHub last year, and I was completely clueless. I didn’t know anything about web development, not even HTML. I stopped asking questions because the answers were too confusing for me. However, I didn’t give up. I studied hard every night and day for the last year and now I am proficient in React.js. I completed every task I set for myself and took detailed notes. Everything I learned is now second nature to me. I learned from trial and error, which I think is the best way to learn. I also took some internationally recognized courses to have some credentials. They helped a bit, but I learned more from watching YouTube videos and practicing on my own. Now, I feel confident in building things with React, but I know I still have a lot to learn. I don’t even know how good I am, because I have been working alone on the basics. I never got any feedback until recently, when someone told me I was in the wrong place. I was in the student section, but I thought that was where I belonged. Now, I am here, and I would love to join a community of developers. Now that I can speak the language, I want to improve my skills and make new friends. My skills are HTML, CSS, Tailwind, JavaScript, and React. I just started learning libraries after building my own components from scratch for a while. I am ready to advance to the next level. Some libraries seem harder than building my own, but I like the idea of using them for the long term. I will build everything in React now to nurture my knowledge of the field. If anyone wants to invite a new guy to their community, I am here every day coding or researching something. I have come to really enjoy it, and this is where I belong.”

**Comment by darkotodoric** [comment]
Well done! You went from knowing nothing about web development to mastering React.js through dedication and self-study. It's great that you're eager to join a community and continue growing your skills. Keep up the good work!

**Reply by MyBaBB** [reply]
"I am currently in the process of learning React and while I have a good understanding of the concept, I still have a long way to go before I can claim mastery. I have spent a considerable amount of time learning the basics of everything before proceeding to work with libraries. It took me a year to feel comfortable enough to start working with them. Contrary to popular belief, learning React in 12 weeks is not an easy feat.

I am here to seek guidance on how to put my basic skills to use on small projects while I continue to build my knowledge. I recently started the beginner course for React on Code Camp to identify any gaps in my knowledge. I understand everything that is being taught and I am confident that it will be a great place for me to learn and perfect my skills."

**Comment by akashdip2001** [comment]
It's fantastic to hear about your journey and the progress you've made in learning web development, especially with React.js. Your dedication and commitment to self-study are truly commendable.

Joining a community is a great step, and it opens up opportunities for collaboration, learning, and networking. Here are a few suggestions to connect with the developer community:

1. **GitHub:**
   - Engage in open source projects on GitHub. It's a great way to contribute to real-world projects and connect with other developers.

2. **Online Communities:**
   - Explore platforms like Stack Overflow, Reddit (e.g., r/reactjs), and Discord channels related to web development.

3. **Meetups and Events:**
   - Attend local or virtual meetups, workshops, and conferences. These events are excellent for networking and gaining insights from experienced developers.

4. **Social Media:**
   - Follow web development communities on Twitter, LinkedIn, and other social platforms. Engage in discussions and share your experiences.

5. **Contribute to Libraries:**
   - Contribute to popular React libraries or create your own. This not only enhances your skills but also allows you to interact with the broader developer community.

6. **Pair Programming:**
   - Collaborate with other developers through pair programming. It's a great way to learn from each other and receive real-time feedback.

7. **Online Courses and Bootcamps:**
   - Consider joining online courses or bootcamps that have community forums. Platforms like Udacity, Coursera, and others often provide spaces for students to interact.

Remember, every developer's journey is unique, and continuous learning is a crucial aspect of staying current in the ever-evolving field of web development. Keep coding, building, and sharing your experiences – you're on the right track!

## Profile
![Profile Image](https://thumbnails.odycdn.com/optimize/s:200:0/quality:95/plain/https://thumbs.odycdn.com/bf58cf4d7ec52c62f02c52e16f150a5b.webp)
**AKASHDIP MAHAPATRA**
"everything is physics bro"  
E=mc²

|[Subscribe on YouTube](https://www.youtube.com/channel/UCxvmp634YDc41xCWOdvWqoQ?sub_confirmation=1) |
|----  |

**Reply by MyBaBB** [reply]
Thank you so much...I like your E=mc2 thing that's the way I always see it too..

**Comment by omarkdro** [comment]


**Reply by MyBaBB** [reply]
Sure, It would help with the experience,  It would be my first actual  project with another person.
Sorry for the long response time, I thought I found an actual job and started working on it.  Turns out it was just an inquiry.   But, I decided to build it anyways because my heart was set on it.  I was already dancing around and singing the song about "How do you like me now." Hollywood sent me the movie script and everything. 
About a guy typing on a computer in the future on a Website.   So, anyways I am making it as my project and they can buy or not. I am almost finished, its simple but I want it to be as nice as possible.   Probably will have to throw it away.   Next time I won't dance around everyone and making a fool of myself.  LOL

**Comment by DarrenChochian** [comment]


**Reply by MyBaBB** [reply]
Hi,   next I figure AI is the route but I have to do this first I figure.   I am 54 So I should probably settle into something soon.  That's why I chose react as my first goal. I am doing it! There is so much to learn. It seems that building something is the best way to learn but I get stuck a lot trying to decide what to learn. 
React Icons was nice to learn and so was material UI.  Can you make a suggestion on a Good React Library to invest time into.?

**Comment by DarrenChochian** [comment]


**Reply by MyBaBB** [reply]
Thank you, I will use what you said and explore those areas now,
I hope that one day I will be able to give such advice.
I feel like I am getting closer.

**Comment by Kavin-Brat** [comment]
Hi @MyBaBB 
Your journey into web development is truly inspiring! You've traveled a long road to become a true master in React.js, and I genuinely appreciate your effort. I'm happy to help you along the way. Best of luck – keep learning, gaining new skills, and building great things!
